### PR TITLE
fix SLE registration without addons

### PR DESCRIPTION
**Pull request by dzedro**
forgot about just SLE registration, too focussed on add-ons ...

### CODE REVIEW

File: lib/registration.pm [hunk 1]
```diff
@@ -69,42 +69,47 @@ sub fill_in_registration_data {
...
-        for $addon (split(/,/, get_var('ADDONS', ''))) {
-            assert_screen("scc-addon-license-$addon");
-            send_key "alt-a", 1;    # accept license
-            send_key "alt-n", 1;    # next
-        }
-        for $addon (split(/,/, get_var('ADDONS', ''))) {
-            $uc_addon = uc $addon;    # change to uppercase to match variable
-            if (my $regcode = get_var("SCC_REGCODE_$uc_addon")) {
-                next if ($addon =~ /sdk|rt/);    # bsc#956726
+        if (get_var('ADDONS')) {
```
Comment (coolo) [inline lib/registration.pm]: someone implemented get_var_array - perhaps you should start using it :)

Comment (dzedro) [inline lib/registration.pm]: heh good news ^^

Comment (coolo) [inline lib/registration.pm]: just checked: Pavel did

**Comment by dzedro** [comment]
Is it ok or should I squash it ?

**Comment by okurz** [comment]
+1, keep it as is is what I say

**Comment by coolo** [comment]
agreed

**Pull request merged**